database/engine/pagecache.c

```diff
@@ -239,7 +239,7 @@ static void pg_cache_release_pages(struct rrdengine_instance *ctx, unsigned numb
 unsigned long pg_cache_hard_limit(struct rrdengine_instance *ctx)
 {
     /* it's twice the number of producers since we pin 2 pages per producer */
```

Comment:
@stelfrag I want to confirm with you that the comment above is still valid, because we are removing the constant that we multiply,  are we sting pinning 2 pages? If we are not, please, remove the comment above and in the next function.